documentation/modules/deploying/proc-deploy-access-operator.adoc

```diff
@@ -0,0 +1,43 @@
+// This assembly is included in the following assemblies:
+//
+// security/assembly-access-operator.adoc
+
+[id='proc-deploy-access-operator-{context}']
+= Deploying the Access Operator
+
+[role="_abstract"]
+This procedure shows how to deploy the Access Operator for access management of a Kafka cluster.
+You cannot use the Access Operator with a Kafka cluster that is not managed by the Cluster Operator.  
+
+.Prerequisites
+
+* The Access Operator installation files must be downloaded and extracted from the latest release archive (`{ReleaseFile}`) from the {ReleaseDownload}. +
```

Comment:
Is the `+` expected at the end of the line?

Reply:
Yes. It puts a soft return in so the next sentence starts underneath.